geopandas/base.py

```diff
@@ -692,6 +692,37 @@ def representative_point(self):
         """
         return _delegate_geo_method("representative_point", self)
 
+    def normalize(self):
+        """Returns a ``GeoSeries`` of normalized geometries to normal form (or canonical form).
+        This method orders the coordinates, rings of a polygon and parts of
```

Comment:
```suggestion
        """Returns a ``GeoSeries`` of normalized geometries to normal form (or canonical form).

        This method orders the coordinates, rings of a polygon and parts of
```

(for docstring formatting, we generally want an emtpy line after the first summary line, and the further description)